### PR TITLE
Fixed LoadingProvider theme token usage in styles

### LINKED ISSUE #903: The `LoadingProvider` component references theme tokens for styling.
### Which package has an issue?

@yamada-ui/react

### Package Version

1.2.8

### Description

Yamada UI allows you to use components without applying a theme. It is a mistake to use theme tokens inside components.

```tsx
const css: CSSUIObject = {
  maxW: 'md',
  display: 'flex',
  flexDirection: 'column',
  justifyContent: 'center',
  alignItems: 'center',
  gap: 'sm',
};
```

There are still some tokens being referenced in the same file.

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### LINKED ISSUE #904: The `NoticeProvider` component references theme tokens for styling.
### Which package has an issue?

@yamada-ui/react

### Package Version

1.2.8

### Description

Yamada UI allows you to use components without applying a theme. It is a mistake to use theme tokens inside components.

```tsx
const css: CSSUIObject = {
  pointerEvents: "auto",
  maxW: "2xl",
  minW: "sm",
  ...style,
}
```

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by suzukisan22**
Closes #903 

## Description

Fixed LoadingProvider theme token usage in styles

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by KenyaMasuko — COMMENTED** [review]
Hi @suzukisan22, thanks your contribute!

There are still some design tokens left in a few places, so please find them and fix them a bit more (e.g., line 429)!

**Comment by suzukisan22** [comment]
Thank you, @KenyaMasuko!
as long as i know, i fixed!! please review!

File: packages/components/loading/src/loading-provider.tsx [hunk 1]
```diff
@@ -426,14 +426,14 @@ const PageComponent = memo(
...
       p: "md",
       display: "flex",
       flexDirection: "column",
       justifyContent: "center",
       alignItems: "center",
-      gap: "sm",
-      rounded: "md",
+      gap: "0.5rem",
+      rounded: "0.375rem",
       boxShadow: ["lg", "dark-lg"],
```
Comment (KenyaMasuko) [inline packages/components/loading/src/loading-provider.tsx:437]: Please refer to `shadows.ts` for reference!
```suggestion
      boxShadow: ["0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)",  "0px 0px 0px 1px rgba(0, 0, 0, 0.1), 0px 5px 10px rgba(0, 0, 0, 0.2), 0px 15px 40px rgba(0, 0, 0, 0.4)"],
```

File: packages/components/loading/src/loading-provider.tsx [hunk 2]
```diff
@@ -474,15 +474,15 @@ const BackgroundComponent = memo(
       position: "fixed",
       right: "md",
       bottom: "md",
```
Comment (KenyaMasuko) [inline packages/components/loading/src/loading-provider.tsx:476]: Please refer to `spaces.ts` for reference!
```suggestion
      right: "1rem",
      bottom: "1rem",
```

File: packages/components/loading/src/loading-provider.tsx [hunk 3]
```diff
@@ -426,14 +426,14 @@ const PageComponent = memo(
   }: LoadingComponentProps) => {
     const css: CSSUIObject = {
       bg: ["white", "black"],
-      maxW: "md",
+      maxW: "24rem",
       p: "md",
```
Comment (KenyaMasuko) [inline packages/components/loading/src/loading-provider.tsx:430]: Please refer to `spaces.ts` for reference!
```suggestion
      p: "1rem",
```

**Review by KenyaMasuko — COMMENTED** [review]
Thanks for the fixes!

There are a few more places that I would like you to revise. I've added some examples of suggested changes in the code, so please take a look and let me know if you have any questions👍

**Comment by suzukisan22** [comment]
@KenyaMasuko Thank you for your comment!! I fixed!!

File: packages/components/loading/src/loading-provider.tsx [hunk 4]
```diff
@@ -358,11 +358,11 @@ const getOverlayStyle = (
   right: 0,
   bottom: 0,
   left: 0,
-  zIndex: "beerus",
+  zIndex: 9999,
   bg: type === "fill" ? ["white", "black"] : "blackAlpha.600",
```
Comment (KenyaMasuko) [inline packages/components/loading/src/loading-provider.tsx:362]: Please refer to `colors.ts` for reference.
```suggestion
  bg: type === "fill" ? ["#fbfbfb", "#141414"] : "rgba(0, 0, 0, 0.48)",
```

File: packages/components/loading/src/loading-provider.tsx [hunk 5]
```diff
@@ -426,15 +426,18 @@ const PageComponent = memo(
   }: LoadingComponentProps) => {
     const css: CSSUIObject = {
       bg: ["white", "black"],
```
Comment (KenyaMasuko) [inline packages/components/loading/src/loading-provider.tsx:428]:
```suggestion
      bg: ["#fbfbfb", "#141414"],
```

File: packages/components/loading/src/loading-provider.tsx [hunk 6]
```diff
@@ -472,18 +475,21 @@ const BackgroundComponent = memo(
   }: LoadingComponentProps) => {
     const css: CSSUIObject = {
       position: "fixed",
-      right: "md",
-      bottom: "md",
-      zIndex: "beerus",
+      right: "1rem",
+      bottom: "1rem",
+      zIndex: 9999,
       bg: ["white", "black"],
```
Comment (KenyaMasuko) [inline packages/components/loading/src/loading-provider.tsx:481]:
```suggestion
      bg: ["#fbfbfb", "#141414"],
```

**Review by KenyaMasuko — COMMENTED** [review]
@suzukisan22 
Sorry I know I'm asking a lot, but could you please make a few more small changes?😅

**Comment by suzukisan22** [comment]
@KenyaMasuko 
No worries!! I just fixed!! please see and check again!

**Review by KenyaMasuko — APPROVED** [review]
@suzukisan22 
I really appreciate all the fixes you made :)
Thanks a lot!!

**Pull request merged**